src/Service.Host/client/src/components/PurchaseOrders/PurchaseOrderUtility.js

```diff
@@ -1198,83 +1197,29 @@ function PurchaseOrderUtility({ creating }) {
                                                                 detail
                                                             )
                                                         }
-                                                        disabled={!allowedToUpdate()}
+                                                        disabled={!creating}
                                                         type="number"
                                                         required
                                                     />
                                                 </Grid>
                                                 <Grid item xs={6}>
-                                                    {overridingOrderQty ? (
```

Comment:
Is there a reason that this has been removed? Surely it's still needed in create mode if the qties are still set here at that time?